### DISCUSSION TITLE
Use pretrained data-driven model to solve the inverse PDE problem

### DISCUSSION THREAD

**Discussion by haison19952013** [lululxvi/deepxde#862] [Q&A]
Dear deepxde community,
I want to ask if there is any way to use a pre-trained data-driven model to solve the inverse problem by using deepxde?
- My current approach is as follows:
  + 1st training step: set the weights for PDE loss functions and Data loss functions to zeros and ones, respectively (purpose: to tune the NN parameters, freeze the PDE parameters)..then make some outputs predictions on the test set with the first trained model
  + 2nd training step: set the weights for PDE loss functions and Data loss functions to ones and zeros, respectively (purpose: to freeze the NN parameters, tune the PDE parameters)..then also make some outputs predictions on the test set with the second trained model
>>  However, this trick does not work because the accuracy of the outputs of the 2 models in step 1 and step 2 is different (i.e., the accuracy of the model in step 2 is very bad). It means the NN parameters are also tuned in step 2, which is not what I want.

**Comment by lululxvi** [comment]
You need to freeze NN parameters in the second step, by modifying a few lines of the source code.
